### PR TITLE
bowerize deparam

### PR DESCRIPTION
**Pull request by chee**
gets us a step closer to no coffee left in rizzo.

this has belonged in vendor 4eva & 5eva

### CODE REVIEW

File: app/assets/javascripts/lib/page/controller.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ define([
   "lib/mixins/events",
   "lib/page/pushstate",
   "lib/mixins/page_state",
-  "lib/utils/deparam"
+  "jquery-bbq-deparam/jquery-deparam"
```
Comment (i-like-robots) [inline app/assets/javascripts/lib/page/controller.js]: should this path be prefixed with `vendor/`?

Comment (i-like-robots) [inline app/assets/javascripts/lib/page/controller.js]: (Or added to RequireJS config)

Comment (chee) [inline app/assets/javascripts/lib/page/controller.js]: no, the vendor/assets are served in the same place as the app/assets

i'm not wrong about this am i?

i'm pretty sure that's right

**Comment by chee** [comment]
@matth  replied

File: app/assets/javascripts/lib/page/controller.js (hunk 1 above)
Comment (i-like-robots) [inline app/assets/javascripts/lib/page/controller.js]: You are right

**Comment by i-like-robots** [comment]
:ship:

**Pull request merged**